### DISCUSSION TITLE
Intune and policy settings

### DISCUSSION THREAD

**Discussion by mooock** [Devolutions/UniGetUI#2890] [Q&A]
Can this software be used with Microsoft Intune, so a policy can be added for which software to allow updating. for deployment i think it will be easy, but more how the default settings could match a company policy.
Will it even work for users not being a local admin. can the service force installs to the user

**Comment by marticliment** [comment]
> Will it even work for users not being a local admin. can the service force installs to the user

For packages that allow such behaviour, yes. However, there is not yet a way to force packages to install in certain settings

**Comment by mooock** [comment]
can i bundle the installer with using only a whitelist it should allow to auto update

**Reply by marticliment** [reply]
Not yet
